src/config.js

```diff
@@ -71,6 +74,16 @@ class Config {
         }
       }
     }
+
+    this.recordedValueMaxLength = coalesce(
+      parseInt(options.recordedValueMaxLength, 10),
+      parseInt(platform.env('SIGNALFX_RECORDED_VALUE_MAX_LENGTH'), 10),
+      DEFAULT_RECORDED_VALUE_MAX_LENGTH
+    )
+    if (this.recordedValueMaxLength < 0) {
+      log.error('Values lower than 0 are not permitted for recordedValueMaxLength.')
```

Comment:
Other libraries allow `<0` and disabled the feature.

Reply:
fixed

Reply:
my only issue now is that suddenly stack traces could start being cut off by default (due to the 1.2 chars limit, which with long paths is not that hard to hit)

Reply:
True but it's the same as every other language :( 

Do we need the log statement still? Why not treat `<0` the same as `-1`?